frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/labeling/SyncToRemoteStatusLabelGenerator.java

```diff
@@ -0,0 +1,25 @@
+package com.virtuslab.gitmachete.frontend.graph.labeling;
+
+import static com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus.Status.Ahead;
+import static com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus.Status.Behind;
+import static com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus.Status.Diverged;
+import static com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus.Status.Untracked;
+
+import java.util.Map;
+
+import com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus;
+
+public final class SyncToRemoteStatusLabelGenerator {
+  private SyncToRemoteStatusLabelGenerator() {}
+
+  private static final Map<SyncToRemoteStatus.Status, String> LABELS = Map.of(
+      Untracked, "untracked",
+      Ahead, "ahead of %s",
+      Behind, "behind %s",
+      Diverged, "diverged from %s");
+
+  @SuppressWarnings("format.string.invalid")
+  public static String getLabel(SyncToRemoteStatus.Status status, String remoteName) {
+    return String.format(LABELS.getOrDefault(status, "sync to %s unknown"), remoteName);
```

Comment:
Hmm I don't think the format is fully correct here - for `"untracked"` the argument is unused (although this likely won't result in a runtime error)